data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://schemas.dissco.tech/schemas/fdo-type/term/0.1.0/term.json",
+  "title": "DiSSCo openDS Term Schema",
+  "description": "A schema for defining standard terms in openDS.",
+  "type": "object",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The unique identifier (handle) of the term object",
+      "pattern": "^https://hdl\\.handle\\.net/[\\w.]+/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "@type": {
+      "type": "string",
+      "description": "The type of the object, in this case ods:Term",
+      "const": "ods:Term",
+      "examples": [
+        "ods:Term"
+      ]
+    },
+    "rdf:type": {
+      "type": "object",
+      "properties": {
+        "@id": {
+          "type": "string",
+          "enum": [
+            "http://www.w3.org/1999/02/22-rdf-syntax-ns#Property",
+            "http://www.w3.org/2000/01/rdf-schema#Class",
+            "http://www.w3.org/2002/07/owl#Class",
+            "http://www.w3.org/2004/02/skos/core#Concept"
+          ]
+        }
+      }
+    },
+    "dcterms:identifier": {
+      "type": "string",
+      "description": "Handle of the Term",
+      "pattern": "^https://hdl\\.handle\\.net/[\\w.]+/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "ods:fdoType": {
+      "type": "string",
+      "description": "The DOI to the FDO type of the object",
+      "pattern": "^https://doi\\.org/[\\w\\.]+/[\\w\\.]+",
+      "examples": [
+        "https://doi.org/21.T11148/bbad8c4e101e8af01115",
+        "https://doi.org/21.T11148/894b1e6cad57e921764e"
+      ]
+    },
+    "ods:status": {
+      "type": "string",
+      "enum": [
+        "ods:Draft",
+        "ods:Active",
+        "ods:Tombstone"
+      ],
+      "description": "The status of the Digital Object",
+      "examples": [
+        "ods:Active"
+      ]
+    },
+    "ods:version": {
```

Comment:
Do we version the terms itself, at the moment we only version the DigitalObject. We could probably do this but don't we need more provenance attributes than? Such as `owl:versionInfo` or `owl:previousVersion` `skos:historyNote`, `dcterms:isVersionOf`, etc...?

Reply:
check tdwg and add previous version

Reply:
add Notes (for decision context)